### DISCUSSION TITLE
Wyze V2 beta fw

### DISCUSSION THREAD

**Discussion by cheme75** [mrlt8/docker-wyze-bridge#380] [Q&A] [answered]
Anyone running the v2 cam beta 4.9.9.1159? It says security improvements- just want to be sure wyze isn’t trying block this bridge.

**Comment by mrlt8** [comment, marked as answer]
Just put 4.9.9.1159 on a spare v2 and can confirm it still works.

**Reply by cheme75** [reply]
Thx! I’m being a bit more prudent about jumping on the latest fw now. With this becoming more popular, I wouldn’t be surprised if Wyze attempts to block it. I also would not be surprised if Wyze is looking at your code to try and fix their web view beta which seems to be in limbo at present. Anyway, I got no idea about how this works exactly, just being cautious like when I had to rely on a jailbreak and unlock to use my iPhone outside the US.
